### DISCUSSION TITLE
Reward Hand of A'dal title for completing BT/Hyjal attunement questlines.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#7091] [Suggestions]
Currently the title is not being rewarded. I propose it be rewarded.

**Comment by Cooties-cc** [comment]
I don't think it's a bad idea, but it can also wait until patch 2.4 drops (or in our case, just when SWP drops). It's "technically" more blizzlike this way, since from patch 2.4 notes (https://wowpedia.fandom.com/wiki/Patch_2.4.0#Dungeons_and_raids) it was given then. 

![image](https://github.com/chromiecraft/chromiecraft/assets/62584589/a9c2597e-b0f4-4534-b1e9-4a8430d76c81)

**Reply by heyitsbench** [reply]
My concern is that the `Champion of the Naaru` title was given with patch 2.3, but on CC it's already been given.

**Comment by elthehablo** [comment]
Looks like this was just booged. The FoS was being handed out, but no title connected to it

**Reply by heyitsbench** [reply]
I know my own character has the `Champion of the Naaru` title but not the Feat of Strength for either attunement line despite completing them both (at least as far as they can be).

**Reply by elthehablo** [reply]
> I know my own character has the `Champion of the Naaru` title but not the Feat of Strength for either attunement line despite completing them both (at least as far as they can be).

That could be because we might've had a similar issue back then, so title was added manually to everyone that completed it? No clue

**Reply by elthehablo** [reply]
But it's weird you don't have the new one either, because it does show up on aowow

**Reply by heyitsbench** [reply]
https://www.chromiecraft.com/en/armory/?character/ChromieCraft/Bench/achievements

See for yourself, no feats of strength listed for either attunements.

**Reply by elthehablo** [reply]
I just queried the db. Nobody has the achievement. Very strange then how we still get the title

**Reply by elthehablo** [reply]
Maybe the FoS reward triggers but it just doesn't get stored for some reason

**Comment by FFabz** [comment]
Hi, will it be retroactive for people who already completed the quest ?

**Reply by elthehablo** [reply]
Yes that is the plan

**Comment by sinhunter** [comment]
I believe it might be connected to the right Vials quest or the wrong version of the quest issue here https://github.com/chromiecraft/chromiecraft/issues/6405
There are characters that have the title already, but most don't.  Not sure if it's client related or the quest id was changed.

**Comment by Nyeriah** [comment]
Hello,

Thanks for your suggestion. It has been accepted.

Regards.
